django_react_proj/users-fe/package.json

```diff
@@ -11,14 +11,19 @@
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.3",
     "antd": "^4.15.1",
+    "apexcharts": "^3.26.1",
     "axios": "^0.21.1",
     "bootstrap": "^4.6.0",
     "date-fns": "^2.21.1",
     "emailjs-com": "^2.6.4",
     "moment": "^2.29.1",
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
+    "react-apexcharts": "^1.3.9",
+    "react-charts": "^2.0.0-beta.7",
     "react-dom": "^17.0.1",
+    "react-minimal-pie-chart": "^8.2.0",
```

Comment:
Are so many chart packages needed?

Reply:
Also any of these added packages are used?

Reply:
The idea was to add a pie chart but i forgot to remove it from package.json